question_thirteen.py

```diff
@@ -0,0 +1,42 @@
+from functools import reduce
+
+class Number:
+    def __init__(self, numbers):
+        self.numbers = numbers
+    
+    def get(self):
+        return self.numbers
+    
+    def change_original_values(self, func: lambda i: i): 
+        new_numbers = []
+        new_numbers = list(map(lambda i: i*2, self.numbers))
```

Comment:
You need to make this dynamic. This is a static one. The values that will be returned will only get multiplied by 2. What If I want to change orginal values to multiple of 3 or 4 or 5, etc.?